core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -27,7 +27,7 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <l:layout title="${it.displayName} Config" cssClass="main-panel-only" norefresh="true" permission="${it.EXTENDED_READ}">
+  <l:layout title="${it.displayName} Config" cssclass="main-panel-only" norefresh="true" permission="${it.EXTENDED_READ}">
     <f:breadcrumb-config-outline />
     <l:main-panel>
```

Comment:
This was [changed](https://github.com/recena/jenkins/blob/JENKINS-31162/core/src/main/resources/hudson/model/Job/configure.jelly#L30). Please, upgrade your repo.